modules/cnf-about-numa-aware-scheduling.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: CONCEPT
+[id="cnf-about-numa-aware-scheduling_{context}"]
+= About NUMA-aware scheduling
+
+NUMA architecture allows a CPU with multiple memory controllers to use any available memory across CPU complexes regardless of where the memory is located. This allows for increased flexibility at the expense of performance. A CPU processing a workload using memory that is outside its NUMA zone is slower than a workload processed in a single NUMA zone. Also, for I/O-constrained workloads, the network interface on a distant NUMA zone slows down how quickly information can reach the application. High-performance workloads, such as telecommunications workloads, cannot operate to specification under these conditions. To process latency-sensitive or high-performance workloads efficiently, all the requested cluster compute resources (CPUs, memory, devices) should be aligned in the same NUMA zone.
+
+The default {product-title} pod scheduler makes scheduling decisions based on available resources of the entire compute node, not individual NUMA zones. If the most restrictive resource alignment is requested in the kubelet topology manager, error conditions can occur when admitting the pod to a node. Conversely, if the most restrictive resource alignment is not requested, the pod can be admitted to the node without proper resource alignment, leading to worse or unpredictable performance. For example, runaway pod creation with `Topology Affinity Error` statuses can occur when the pod scheduler makes suboptimal scheduling decisions for guaranteed pod workloads by not knowing if the pod's requested resources are available. Scheduling mismatch decisions can cause indefinite pod startup delays. Also, depending on the cluster state and resource allocation, poor pod scheduling decisions can cause extra load on the cluster because of failed startup attempts.
+
+The NUMA Resources Operator deploys a custom NUMA resources secondary scheduler and other resources to mitigate against the shortcomings of the default {product-title} pod scheduler. An overview of NUMA-aware scheduling with the NUMA resources secondary scheduler is described below.
```

Comment:
I have been informed that we should not use below in docs 
! below adjective, adverb, preposition
Do not use to indicate a relative location in a document, as in "the information below".